packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import uuid
+
+import requests
+
+from test_util.marathon import get_test_app
+
+
+def test_if_marathon_app_can_be_debugged(cluster):
+    def post(url, headers, data, stream=False):
+        r = requests.post(url, headers=headers, json=data, stream=stream)
+        logging.info(
+            'Got %s with POST request to %s with headers %s and data %s.',
+            r.status_code,
+            url,
+            headers,
+            data
+        )
+        assert r.status_code == 200
+        return r
+
+    def find_container_id(state, app_id):
```

Comment:
Agreed. +1 to code reuse. I actually copied most of this from the cli code anyway. :)